packages/backend/src/migration/1605695145000-posOnExistingData.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class posOnExistingData1605695145000 implements MigrationInterface {
+    name = 'posOnExistingData1605695145000'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`UPDATE "modos"."observation" SET "updated_at"=NOW();`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" ALTER COLUMN "position" SET NOT NULL;`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" ALTER COLUMN "edist" SET NOT NULL;`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" ALTER COLUMN "eid" SET NOT NULL;`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."observation" ALTER COLUMN "snap_geom" SET NOT NULL;`, undefined);
```

Comment:
If we got a mix of camel and snake case in a table, it will the begining of problem. Stick with CamelCase for this table.

Reply:
Won't fix now.